dotnet-logging/LogLevelsWithSerilog/Tests/HomeControllerTests.cs

```diff
@@ -11,102 +6,72 @@ public class LogLevelsWithSerilogTests
     private HomeController? _homeController;
     private Mock<ILogger<HomeController>>? _loggerMock;
     private IFixture? _fixture;
+    private string _message = "Serilog Test Message";
```

Comment:
We can just initialize this to an empty string.

Reply:
fixed